sbt-scala-native-crossproject/src/main/scala/scalanativecrossproject/ScalaNativeCrossPlugin.scala

```diff
@@ -0,0 +1,22 @@
+package scalanativecrossproject
+
+import sbt._, Keys._
+import scalanative.sbtplugin._
+import org.portablescala.sbtplatformdeps.PlatformDepsPlugin.autoImport._
+
+import scala.language.implicitConversions
+
+object ScalaNativeCrossPlugin extends AutoPlugin {
+  override def trigger           = allRequirements
+  override def requires: Plugins = ScalaNativePlugin
+
+  object autoImport extends ScalaNativeCross
+
+  import autoImport._
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    platformDepsCrossVersion := ScalaNativeCrossVersion.binary,
```

Comment:
This is not necessary, as [it is already provided by sbt-scala-native](https://github.com/scala-native/scala-native/blob/682dab82c7c689d00ebe6d263685694994287ad4/sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala#L54). This also means that there is no need for the `def trigger` and `def requires` (once the `resolvers` thing is gone, of course). In fact, this `AutoPlugin` is only used for its `autoImport` :p

Note that in sbt-scalajs-crossproject, the story is different, because Scala.js 0.6.x does not set `platformDepsCrossVersion`, so we need that setting for legacy reasons. That will change once we publish sbt-platformdeps 1.0.0, at which point we'll be able to use sbt-platformdeps in Scala.js 0.6.x as well and hence set it from sbt-scalajs: https://github.com/scala-js/scala-js/pull/3166/files#diff-dd593403434ec9de3670cb5722f10171.